recipes/cuda-api-wrappers/config.yml

```diff
@@ -1,5 +1,15 @@
 versions:
+  "0.7-b2":
+    folder: all
   "0.7-b1":
     folder: all
+  "0.6.8":
+    folder: all
+  "0.6.7":
+    folder: all
+  "0.6.6":
+    folder: all
+  "0.6.4":
+    folder: all
```

Comment:
```suggestion
  "0.6.8":
    folder: all
```

Reply:
Done by revising the first commit on this branch.